packages/grafana-ui/src/components/ButtonCascader/_ButtonCascader.scss

```diff
@@ -1,6 +1,15 @@
 .rc-cascader {
   font-size: 12px;
 
+  &-dropdown {
+    // Required, otherwise the portal that the popup is shown in will render under other components
+    z-index: 9999;
```

Comment:
![my eyes](https://media.giphy.com/media/MeDlAyl2m0lhrGRgCm/giphy.gif)

I understand why we need it but feels like going down a rabbit hole when starting to introduce things like this.

Reply:
@mckn hahaha, i totally agree, but i didn't introduce it 😄  just moved it from here: https://github.com/grafana/grafana/pull/41470/files#diff-08368c9249935c21e12683080f8e5b516060b89adf4fd18771e3f63ed8d85a0aL215-L218